lib/plugins/timer/index.js

```diff
@@ -3,46 +3,32 @@ import parseTiming from './parse-timing';
 export default shower => {
     let id;
 
-    const goNext = () => {
-        shower.player.next();
-    };
+    const { player } = shower;
+    const setTimer = () => {
+        clearTimeout(id);
+        if (shower.isListMode) return;
```

Comment:
Seems that we still have the magical `shower` as a God object. I think we should avoid passing that around; the player should know what the mode is.